src/AuthModule/Components/Register/Register.tsx

```diff
@@ -1,11 +1,286 @@
-import React from 'react'
+import axios from "axios";
+import { useContext, useState } from "react";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import Loader from "../../../SharedMoudule/Components/Loading/Loading";
+import { countryValidation, emailValidation, passwordValidation, phoneNumberValidation, userNameValidation } from "../../../SharedMoudule/Components/Validator/Validator.js";
+import defaultImage from "../../../assets/Avatar.png";
+import logo from "../../../assets/PMS 3.svg";
+import styles from "./Register.module.css";
+import { baseUrl } from "../../../Constants/Components/Urls.js";
 
 export default function Register() {
+  const [userImage, setUserImage] = useState("")
+
+  const [showPassword, setShowPassword] = useState(true);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(true);
+  const [isLoading, setIsLoading] = useState(false);
+  const navigate = useNavigate();
+
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    watch,
+  } = useForm();
+  const confirmPassword = watch("password");
+
+  async function handleRegister(values: any) {
+    try {
+      const { data } = await axios.post(`${baseUrl}/Users/Register`, values);
+      console.log(data);
+      toast.success("Account created successfully.");
+      navigate("/VerifyAccount");
+    } catch (error: any) {
+      console.log(error);
```

Comment:
While logging to the console is crucial but you need to delete it when you prepare your code for production, for better performance and to not expose any sensitive information to the user.